### DISCUSSION TITLE
Kullandığın iterm teması nedir ?

### DISCUSSION THREAD

**Discussion by onuryilmazdev** [ademilter/homepage#36] [Q&A] [answered]
Videolarında kullandığın terminal iterm sanırım bunda kullandığın tema nedir?

**Comment by ademilter** [comment, marked as answer]
Merhaba Onur,

- [iTerm2](https://www.iterm2.com/)
- [ohmyz](https://ohmyz.sh/)
- [theme snazzy](https://github.com/sindresorhus/iterm2-snazzy)
- [JetBrains Mono typeface](https://www.jetbrains.com/lp/mono/)
